### PR TITLE
Exemple fixing

### PR DESCRIPTION
**Pull request by giovannicampagnolo**
Fixing all the example with updating null-safety and fix also scaling on ios models

### CODE REVIEW

**Review by nsvetlakov — APPROVED** [review]

**Comment by giovannicampagnolo** [comment]
@CariusLars

File: example/pubspec.yaml [hunk 1]
```diff
@@ -6,11 +6,16 @@ description: Demonstrates how to use the ar_flutter_plugin plugin.
...
+  sdk: ">=2.16.1 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
+  permission_handler: 10.1.0
+  vector_math: 2.1.1
+  json_annotation: 4.5.0
+  geolocator: 9.0.0
+  app_settings: 4.1.6
```
Comment (CariusLars) [inline example/pubspec.yaml:18]: why did you add these dependencies?

**Review by CariusLars — CHANGES_REQUESTED** [review]
Added a comment. Two other points:

1. If you adapt the scaling on iOS, isn't the duck in the example application way bigger than on android now?
2. could you add an entry in changelog.md for your patch version and update the version referenced in the readme.md, please?

Thanks a lot for your contribution!

**Comment by giovannicampagnolo** [comment]
ok @CariusLars then the scaling 0.01 is for duck now i fix

**Comment by giovannicampagnolo** [comment]
i made the request fix if you want to check @CariusLars

File: example/pubspec.yaml (hunk 1 above)
Comment (giovannicampagnolo) [reply example/pubspec.yaml]: it is my error now i fix

File: ios/Classes/ArModelBuilder.swift [hunk 2]
```diff
@@ -126,7 +126,7 @@ class ArModelBuilder: NSObject {
             scene = try sceneSource.scene()
 
             for child in scene.rootNode.childNodes {
-                child.scale = SCNVector3(0.01,0.01,0.01) // Compensate for the different model dimension definitions in iOS and Android (meters vs. millimeters)
+                child.scale = SCNVector3(1,1,1) // Compensate for the different model dimension definitions in iOS and Android (meters vs. millimeters)
```
Comment (CariusLars) [inline ios/Classes/ArModelBuilder.swift:129]: can you revert this as well please?

File: pubspec.yaml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-version: 0.7.2
+version: 0.7.3
 homepage: https://lars.carius.io
 repository: https://github.com/CariusLars/ar_flutter_plugin
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.20.0"
+  sdk: ">=2.16.1 <3.0.0"
+  #flutter: ">=1.20.0"
```
Comment (CariusLars) [inline pubspec.yaml:9]: why did you comment this out?

**Review by CariusLars — CHANGES_REQUESTED** [review]
thanks for the updates! two more small comments left in the code and could you please also update the readme to reference your new version 0.7.3 in the installation section?

File: pubspec.yaml (hunk 3 above)
Comment (giovannicampagnolo) [reply pubspec.yaml]: i fixed also this

File: ios/Classes/ArModelBuilder.swift (hunk 2 above)
Comment (giovannicampagnolo) [reply ios/Classes/ArModelBuilder.swift]: fix

**Comment by giovannicampagnolo** [comment]
@CariusLars i resolve the last issues 

Thank you

**Comment by CariusLars** [comment]
merged it, thanks a lot for your contribution @giovannicampagnolo !

**Pull request merged**